aserver/worker.py

```
@@ -0,0 +1,78 @@
+    postgres_fingerprint_dsn: str,
+    postgres_ingest_dsn: str,
+    postgres_musicbrainz_dsn: str,
+) -> None:
+    asyncio.run(
+        worker(
+            nats_servers=list(nats_servers),
+            postgres_app_dsn=postgres_app_dsn,
+            postgres_fingerprint_dsn=postgres_fingerprint_dsn,
+            postgres_ingest_dsn=postgres_ingest_dsn,
+            postgres_musicbrainz_dsn=postgres_musicbrainz_dsn,
+        )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)